bot.py

```diff
@@ -82,19 +86,13 @@ def get_subscriptions(self):
         self.send(subscriptions_msg)
 
     def subscribe(self, channel, channel_id):
-        subscribe_msg = {
```

Comment:
Is there any way to not change the formatting on these messages?

Reply:
No, ["black is not configurable"](https://black.readthedocs.io/en/stable/the_black_code_style.html).